core/src/main/java/org/testcontainers/utility/ResourceReaper.java

```diff
@@ -125,7 +135,23 @@ public void onNext(Frame frame) {
 
         ContainerState containerState = new ContainerState() {
 
-            final InspectContainerResponse inspectedContainer = client.inspectContainerCmd(ryukContainerId).exec();
+            // inspect container response might initially not contain the mapped port
+            final InspectContainerResponse inspectedContainer = await()
+                .atMost(5, TimeUnit.SECONDS)
+                .pollInterval(DynamicPollInterval.ofMillis(10))
```

Comment:
now that we support fixed intervals, perhaps we should not be using `10ms`, but something less aggressive (e.g. `50ms`), as, if the ports aren’t mapped, some syscalls are involved and I doubt the second attempt (after 10ms) will ever succeed.

WDYT?

Reply:
right, `50ms` seem sensible, will change it in both locations.

Reply:
FWIIW, 5s as failure timeout might also be unnecessary high, but I don't have anything to back up what a good maximum might be (maybe 2s?).

Reply:
5s is okay I think - not too much, while 2s feels a bit too small :D